src/test/vat.t.sol

```diff
@@ -672,18 +633,21 @@ contract BiteTest is DSTest {
 
         // tag=4, mat=2
         vat.file("gold", 'spot', ray(1 ether));  // now unsafe
+        // cat.file("gold", "chop", 1.1 ether);
 
         assertEq(ink("gold", address(this)), 100 ether);
         assertEq(art("gold", address(this)), 150 ether);
         assertEq(vow.Woe(), 0 ether);
         assertEq(gem("gold", address(this)), 900 ether);
 
         cat.file("box", rad(75 ether));
-        cat.file("gold", "lump", rad(100 ether));
+        cat.file("gold", "dunk", rad(100 ether));
         assertEq(cat.box(), rad(75 ether));
         assertEq(cat.litter(), 0);
         uint auction = cat.bite("gold", address(this));
-        assertEq(cat.litter(), rad(75 ether));
+        // (,,,,,,,uint tab) = flip.bids(auction);
+        // assertTrue(rad(75 ether) - cat.litter() < ray(3 ether));
+        // assertEq(cat.litter(), tab);
```

Comment:
Why is this commented out?

Reply:
These are the tests that we started playing with where we set chop to a non-zero percentage.  This is how we found the dusty remainder, and eventually the dusty auctions, but it was not directly responsible for testing for dust.

I should just remove these comments, but perhaps there is still some desire to make a realistic value tests with a 10% `chop`?